gammapy/analysis/config.py

```diff
@@ -134,6 +134,13 @@ class FluxPointsConfig(GammapyBaseConfig):
     parameters: dict = {"selection_optional": "all"}
 
 
+class LightCurveConfig(GammapyBaseConfig):
+    time_intervals: TimeRangeConfig = TimeRangeConfig()
+    energy_edges: EnergyAxisConfig = EnergyAxisConfig()
```

Comment:
Consider that in the [excess map PR](https://github.com/gammapy/gammapy/pull/3307) the `EnergyAxisConfig` class has now `None` values by default. Just in case, maybe you would like to add default values for the `energy_edges` as you did in the other PR for `energy` and `energy_true`..

Reply:
The default for the `LightCurveEstimator` is to use `None` for the default `energy_edges` (i.e. take the full valid range of the data). I suppose we should stick to this default.